api-gateway/src/main/java/com/petclinic/bffapigateway/presentationlayer/v2/CartController.java

```diff
@@ -30,7 +30,8 @@ public class CartController {
 
     private final CartServiceClient cartServiceClient;
 
-    @SecuredEndpoint(allowedRoles = {Roles.ADMIN})
+    //later we will need to check if the user that is logged in isnt getting someone else's carts, but thats for sprint 3
```

Comment:
Have you added the task  to Jira?

Reply:
no, but thank you for pointing that out! i'll put it in the backlog